niv-rozanowich/style.css

```diff
@@ -0,0 +1,1500 @@
+@font-face {
```

Comment:
it's a better practice to separate the font styles inside a directory, and not to put it on the main folder of your project.